sdk/communication/AzureCommunicationChat/Tests/PolicyTokenCredentialTests.swift

```diff
@@ -24,23 +24,19 @@
 //
 // --------------------------------------------------------------------------
 
+import AzureCommunication
+import AzureCommunicationChat
+import AzureCore
 import XCTest
 
-#if canImport(AzureCommunication)
```

Comment:
Why you remove the #if canImport?

Reply:
Technically these were tests, it never really needed them in the first place.